### ISSUE TITLE
More informative team descriptions

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/team#334]
Many teams on @rust-lang have a description (the extra text shown when I type `@rust-lang/`) that just says "managed by the team repository". That's not very useful when I want to figure out which is the right team to ping. Would be better to instead show a brief description of what the team is doing. :)

**Comment by RalfJung** [comment]
FWIW, as more and more teams are converted to be managed by this repo, we are losing even the useful descriptions that we used to have.

**Comment by Mark-Simulacrum** [comment]
We don't currently have description metadata outside of the website section in the toml configs here, and I'd rather not silently use that in other places.

Other than that problem, https://github.com/rust-lang/sync-team/blob/master/src/github/mod.rs#L69 is the place to edit to put more descriptive descriptions in place.

**Comment by rylev** [comment]
We can add a description section to the toml configuration for the GitHub team, and sync that if available.

**Comment by RalfJung** [comment]
> We don't currently have description metadata outside of the website section in the toml configs here, and I'd rather not silently use that in other places.

Hm, that description seems fine for me (looking at the lang team, at least) -- why would one not want to also use that in other places?

**Comment by rylev** [comment]
I think the issue is that that description is explicitly inside the metadata *for the website*, and this can become confusing if it actually gets used elsewhere.

**Comment by Mark-Simulacrum** [comment]
Yeah, I think having a generic description we (also) use for the website is ok, but using the website description implicitly seems likely to confuse us later down the line (e.g., removing a website section causes GitHub description to disappear).
